db/migrate/20181001142852_create_topics.rb

```diff
@@ -2,6 +2,8 @@ class CreateTopics < ActiveRecord::Migration[5.2]
   def change
     create_table :topics do |t|
       t.string :label, null: false
+
+      t.timestamps
```

Comment:
You should not change a migration. If you need to change something, use a new migration.